libs/go/zmscli/dump.go

```diff
@@ -447,10 +447,12 @@ func (cli Zms) dumpObjectList(buf *bytes.Buffer, list []string, dn string, objec
 	}
 }
 
-func (cli Zms) dumpServices(buf *bytes.Buffer, dn string) {
+func (cli Zms) dumpServices(buf *bytes.Buffer, dn string, tagKey string, tagValue string) {
+	buf.WriteString(indentLevel1)
+	buf.WriteString("services:\n")
```

Comment:
I am pretty sure we cannot add here texts, since there are teams that are parsing zms-cli output.
@havetisyan is it correct?

anyway, why did you add it?

Reply:
show services use this function in order to show all the services return from the query.